config/example/node.yaml

```diff
@@ -1,159 +1,159 @@
 
-contracts:
+contracts:  # side chain NEOFS contract script hashes
   balance: 5263abba1abedbf79bb57f3e40b50b4425d2d6cd
   container: 5d084790d7aa36cea7b53fe897380dab11d2cd3c
   netmap: 0cce9e948dca43a6b592efe59ddb4ecb89bdd9ca
   reputation: 441995f631c1da2b133462b71859494a5cd45e90
   proxy: ad7c6b55b737b696e5c82c85445040964a03e97f
 
 morph:
-  dial_timeout: 30s
-  disable_cache: true
-  rpc_endpoint:
+  dial_timeout: 30s  # timeout for side chain NEO RPC client connection
+  disable_cache: true  # do not use TTL cache for side chain GET operations
+  rpc_endpoint:  # side chain NEO RPC endpoints; are shuffled and used one by one until the first success
     - https://rpc1.morph.fs.neo.org:40341
     - https://rpc2.morph.fs.neo.org:40341
-  notification_endpoint:
+  notification_endpoint:  # side chain NEO RPC notification endpoints
     - wss://rpc1.morph.fs.neo.org:40341/ws
     - wss://rpc2.morph.fs.neo.org:40341/ws
   notary:
-    deposit_amount: 100000001
-    deposit_duration: 1001
+    deposit_amount: 100000001  # notary deposit GAS amount
+    deposit_duration: 1001  # notary deposit expiration in blocks
 
 mainchain:
-  dial_timeout: 30s
-  rpc_endpoint:
+  dial_timeout: 30s  # timeout for main chain NEO RPC client connection
+  rpc_endpoint:  # main chain NEO RPC endpoints; are shuffled and used one by one until the first success
     - https://rpc1.n3.nspcc.ru:30341
     - https://rpc2.n3.nspcc.ru:30341
 
 apiclient:
-  dial_timeout: 15s
+  dial_timeout: 15s  # timeout for NEOFS API client connection
 
 policer:
-  head_timeout: 15s
+  head_timeout: 15s  # timeout for the Policer HEAD remote operation
 
 replicator:
-  put_timeout: 15s
+  put_timeout: 15s  # timeout for the Replicator PUT remote operation
 
 object:
   put:
-    pool_size_remote: 100
-    pool_size_local: 101
+    pool_size_remote: 100  # numbers of async workers for remote PUT operations
+    pool_size_local: 101  # numbers of async workers for local PUT operations
 
 storage:
-  shard_num: 2
+  shard_num: 2  # total shards amount
   shard:
     0:
-      refill_metabase: false
+      refill_metabase: false  # sync metabase with blobstore on start, expensive, leave false until complete understanding
 
-      use_write_cache: false
+      use_write_cache: false  # use write-cache
 
       writecache:
-        path: tmp/0/cache
-        mem_size: 2147483648
-        small_size: 16384
-        max_size: 134217728
-        workers_number: 30
-        size_limit: 3221225472
+        path: tmp/0/cache  # write-cache root directory
+        mem_size: 2147483648  # coarse RAM usage limit for "small" objects
+        small_size: 16384 # size threshold for objects that are considered to be "small"
+        max_size: 134217728  # objects that are bigger than <max_size> bypass write-cache and go to the storage directly
+        workers_number: 30  # thread number of write-cache flushers
+        size_limit: 3221225472  # coarse write-cache total size
```

Comment:
units (bytes) should be specified

Reply:
added ", bytes" suffix in every mem-size-config values